switchboard/conditions.py

```diff
@@ -11,11 +11,12 @@
 
 import datetime
 import itertools
+import re
 
 from .models import EXCLUDE
 
 
-class Invalid(Exception):
+class Invalid(Exception):  # pragma: nocover
```

Comment:
But why? It would be creating a unit test *just* for the purpose of coverage, not because it actually *tests* anything meaningful.